frontend/wallet/schema.graphql

```diff
@@ -73,6 +73,12 @@ enum ChainReorganizationStatus {
   CHANGED
```

Comment:
This file is generated now, by running the daemon with rest port 8080, going to coda/frontend/wallet and running `yarn regen-sdl`

Reply:
Damm, I love this thing. Thank you